src/service/extension-location.js

```diff
@@ -39,18 +39,11 @@ function calculateScriptBaseUrl(location, isLocalDev, isTest) {
  * @param {string} extensionId
  * @param {boolean=} isLocalDev
  * @param {boolean=} isTest
- * @param {boolean=} isUsingCompiledJs
  * @return {string}
  */
 export function calculateExtensionScriptUrl(location, extensionId, isLocalDev,
-    isTest, isUsingCompiledJs) {
+    isTest) {
   const base = calculateScriptBaseUrl(location, isLocalDev, isTest);
```

Comment:
I realize it's not possible to delete `calculateScriptBaseUrl`. Will leave as it is.